### ISSUE TITLE
Disabling plugin in blender result in blender crash

### ISSUE THREAD

**Issue by radicalaction** [heisenshark/blender-krita-link-plugin#14]
![unregister](https://github.com/user-attachments/assets/e1d76d83-cac8-44a9-a7ce-477921bda807)
Disabling plugin on the blender side crashes blender. First this popup appears and after a couple of seconds blender crash. Clicking "Uninstall" in the addons menu doesn't do anything except crashing blender as well.

**Comment by heisenshark** [comment]
I did not test it in the newest blender version, right now its quick fix in BlenderKritaLink/__init__.py 
if you want fast fix

```python
# replace lines 113-114 to this 
    if bpy.app.timers.is_registered(UvWatch.instance.check_for_changes):
        bpy.app.timers.unregister(UvWatch.instance.check_for_changes)
    if bpy.app.timers.is_registered(ImagesStateWatch.instance.check_for_changes):
        bpy.app.timers.unregister(ImagesStateWatch.instance.check_for_changes)
```

**Comment by radicalaction** [comment]
@heisenshark Thanks, it worked in the latest blender version as well.

**Comment by heisenshark** [comment]
fixed in [0.3](https://github.com/heisenshark/blender-krita-link-plugin/releases/tag/0.3)
